### ISSUE TITLE
Hardware Wallet Creation

### ISSUE THREAD

**Issue by gdelgado** [Berry-Pool/nami-wallet#112]
Currently the user first needs to create a new wallet and then will have the option to create a new account via a hardware wallet.  The interface should allow the user to create a new account via a hardware wallet without the need to create an initial seed phrase wallet.

**Comment by pascallapointe** [comment]
Yes this should be the behavior and we would like to make it this way somewhere in the future.

This will require a lot of change to the internal structure of the wallet.
The current behavior was the fastest way to make the hardware wallet feature available without going in a feature freeze for some months.
